modules/viewing-the-details-of-a-pipeline-run.adoc

```diff
@@ -24,12 +24,15 @@ endif::[]
 The *Pipelines* page opens.
 . From the *Project* list, select the project whose pipeline runs you want to view.
 . For a pipeline that you want to see run details for, click *Expand* (image:images/rhoai-expand-icon.png[]).
```

Comment:
Could change 'see' to 'view' to be consistent.

Reply:
Fixed.